tests/plugins/inventory/helpers/test_filter.py

```diff
@@ -0,0 +1,43 @@
+from nornir.plugins.inventory.helpers.filters import F
```

Comment:
As per above comment this should be "tests/core/test_filter.py".

Also, please, follow the same pattern as with the rest of the tests, please.